### ISSUE TITLE
Event IDs incorrectly associated with log source Providers 

### ISSUE THREAD

**Issue by JPvRiel** [JPvRiel/wef-reference#1]
As an example, the "Spooler" Provider has a link to the "Security" log path, but does not define any event IDs. The 'query_combinations_flattened_by_event.csv" can incorrectly associate event IDs, like event ID 4624, with the Spooler service if given a Select XPath such as `EventID=4624`. E.g. compare two files in custom and run `./compare_wef.py -c`

Provider specified:

```xml
<QueryList>
  <Query Id='0'>
    <Select Path="Security">
        *[System[Provider[@Name='Microsoft-Windows-Security-Auditing'] and EventID=4624]]
    </Select>
  </Query>
</QueryList>
```

Provider not specified:

```xml
<QueryList>
  <Query Id='0'>
    <Select Path="Security">
        *[System[EventID=4624]]
    </Select>
  </Query>
</QueryList>
```

The second one without the provider associates event IDs with "Spooler" and other Providers which didn't have event IDs defined in their manifests.

**Comment by JPvRiel** [comment]
Closed by 7a742af

**Comment by JPvRiel** [comment]
Closed by 7a742af
